main.go

```diff
@@ -215,6 +217,7 @@ func mountWithArgs(
 		bucketName,
 		mountPoint,
 		flags,
+		mountConfig,
```

Comment:
Not sure on this but mountConfig sounds like (flags + bucketName + mountPoint). So, naming it like configFileFlags or configFileOptions might be better.

Reply:
configFile doesn't sound appropriate. I went back and forth on the naming mountOptions, mountConfig etc. Went with mountConfig just because it is the config specified at mount time. 

Let me know if you have any other suggestion.

Reply:
Even I don't have any other suggestion.